recipes/fbgemm/all/conanfile.py

```diff
@@ -0,0 +1,92 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd, check_min_cstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+import os
+
+required_conan_version = ">=2.4"
+
+
+class FbgemmConan(ConanFile):
+    name = "fbgemm"
+    description = (
+        "Facebook General Matrix Multiplication is a low-precision, high-performance matrix-matrix multiplication"
+    )
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/pytorch/FBGEMM"
+    topics = ("convolution", "matrix", "multiplication", "quantization", "performance")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    implements = ["auto_shared_fpic"]
+    languages = "C", "C++"
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("cpuinfo/[>=cci.20250321]", transitive_headers=True, transitive_libs=True)
+        self.requires("asmjit/cci.20241216", transitive_headers=True, transitive_libs=True)
+
+    def validate(self):
+        check_min_cppstd(self, 17)
+        if self.settings.get_safe("compiler.cstd"):
+            check_min_cstd(self, 99)
+        if self.settings.arch == "x86":
+            raise ConanInvalidConfiguration("fbgemm is not supported on x86 architectures")
+        if "arm" in self.settings.arch:
+            raise ConanInvalidConfiguration("fbgemm is not supported on ARM")
```

Comment:
fbgemm has support (limited) for arm, I think the problem is that the test_package is using operations that are not compatible with arm.